public/js/connectivity-manager.js

```diff
@@ -49,8 +43,8 @@ export default class Offline {
    * @param url the url to be checke for availability
    * @returns true if the user is online or the URL is cached
    */
-  static isAvailable(href) {
-    if (!href || window.navigator.onLine) return Promise.resolve(true);
+  isAvailable(href) {
```

Comment:
Since this exists, remove the [`toolbox.router.head(.)` route](https://github.com/GoogleChrome/gulliver/blob/master/public/sw.js#L61-L80)? I don't think it's needed.